procedures/MT/rotator/rotator_performance.ipynb

```diff
@@ -0,0 +1,655 @@
+{
```

Comment:
<p>This is confusing because you run the same code three times... so it's not obvious why this checks the negative position first... </p><p>Maybe the description above will help?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
Fixed in the commit: [30d19d5](https://github.com/lsst-ts/ts_notebooks/pull/96/commits/30d19d514c64981689c466b89131d18ff78a1464). Thanks!